src/installer/pkg/packaging/rpm/pack-deps.proj

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Can just use `<Project>`, extra stuff up here is legacy now.

Reply:
Will fix.

Reply:
Removing this whole proj file.